plugins/askrene/reserve.c

```diff
@@ -118,6 +119,29 @@ void reserve_sub(const struct reserve_htable *reserved,
 	}
 }
 
+void json_add_reservations(struct json_stream *js,
+			   const struct reserve_htable *reserved,
+			   const char *fieldname)
+{
+	struct reserve *r;
+	struct reserve_htable_iter rit;
+
+	json_array_start(js, fieldname);
+	for (r = reserve_htable_first(reserved, &rit);
+	     r;
+	     r = reserve_htable_next(reserved, &rit)) {
+		json_object_start(js, NULL);
+		json_add_short_channel_id_dir(js,
+					      "short_channel_id_dir",
+					      r->scidd);
+		json_add_u64(js, "age_in_seconds",
+			     timemono_between(time_mono(), r->timestamp).ts.tv_sec);
```

Comment:
Small optimization: make a single call to `time_mono()` before the loop. Use that value as the present time
and use that to compute the age.